scripts/kvm-test.py

```diff
@@ -43,6 +42,22 @@
 '''
 
 
+@dataclasses.dataclass
+class Profile:
+    name: str
+    memory: str
+    disk_size: str
+    extra_qemu_options: list[str]
+
+
+profiles = {
+
+    'server': Profile(name='server', memory='2G', disk_size='12G', extra_qemu_options=[]),
```

Comment:
@dbungert you had previously set the default memory to 8 GiB. Would 2 GiB work for you? I can bump it up if needed.

Reply:
2GB is fine for a default, but this breaks the config setup, so it'd be nice to be able to specify the profiles in the yaml cfg.  Do you have thoughts on that part?

Reply:
I expected that this would be a request, yes :)

I've pushed a change allowing to override a profile (or to create a new one) using:

```yaml
iso:
   # [...]
profiles:
  server:
    # server already exists in the default config, so we do not need to specify all the settings
    memory: 4G

  server-cpu4-ram16-disk20
    # this will create a new profile, all the properties must be specified
    memory: 16G
    qemu-extra-options: [-smp, "4"]
    disk-size: 20G
```

does it help?